docs/cql/ddl.rst

```diff
@@ -165,8 +165,16 @@ sub-option                             type  description
                                              definitions or explicit datacenter settings.
                                              For example, to have three replicas per
                                              datacenter, supply this with a value of 3.
+``'tablets'``                          bool  Whether or not to use tablets replication for
+                                             this keyspace
+``'initial_tablets'``                  int   The number of tablets each table starts with.
+                                             Applies only if `tablets=true`. If missing,
+                                             the number is selected automatically based on
+                                             datacenter refplication factors
```

Comment:
I tried to document initial_tablets in https://github.com/scylladb/scylladb/pull/16193